modules/manage/pages/kubernetes/k-manage-resources.adoc

```diff
@@ -283,7 +303,9 @@ If you use PersistentVolumes, you can set the storage capacity for each volume.
 
 If Redpanda runs in a shared environment, where multiple applications run on the same worker node, you can make Redpanda less aggressive in CPU usage by enabling overprovisioning. This adjustment ensures a fairer distribution of CPU time among all processes, improving overall system efficiency at the cost of Redpanda's performance.
 
-You can enable overprovisioning by either setting the CPU request to a fractional value or setting `overprovisioned` to `true`.
+You can enable overprovisioning by either setting the CPU request to a fractional value or setting `resources.cpu.overprovisioned` to `true`.
```

Comment:
nit: clearify that fractional must be < 1 for overprovisioned to be set
```suggestion
You can enable overprovisioning by either setting the CPU request to a fractional value less than 1 or setting `resources.cpu.overprovisioned` to `true`.
```

And similarly to the comment above, overprovisioned can be set with either a CLI flag or an RPK config as well. Let's align on either CLI flags or RPK config values for this as well.